File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// N4chan SDK exists test

require_once __DIR__ . '/../n4chan_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = N4chanSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: rb/test/exists_test.rb
# N4chan SDK exists test

require "minitest/autorun"
require_relative "../N4chan_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = N4chanSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
